tests/conftest.py

```diff
@@ -283,17 +283,35 @@ def init_kwargs(request):
 
 
 @pytest.fixture(scope="function")
-def task(request, init_kwargs):
+def task(request, generate_task):
     """Gets a task instance."""
```

Comment:
For both `task` and `generate_task`, add a warning about ownership like
https://github.com/ni/nidaqmx-python/blob/master/tests/unit/conftest.py#L26

Reply:
updated as per suggestion